build/Dockerfile

```diff
@@ -1,8 +1,20 @@
+FROM openshift/origin-release:golang-1.10
+
+RUN mkdir -p /go/src/github.com/embercsi/ember-csi-operator/
+RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
Why do we need to do this installation?

Reply:
This is the dependency tool executed by "make dep"

Reply:
Using an sh script from master seems a little feeble, but I guess it's OK for now.